src/postprocessing.py

```diff
@@ -37,6 +77,16 @@ def categorize_image(image):
     return np.argmax(image, axis=0)
 
 
+def categorize_multilayer_image(image):
+    categorized_image = []
+    for category_id, category_output in enumerate(image):
+        thrs_step = 1. / (CATEGORY_LAYERS[category_id] + 1)
```

Comment:
@jakubczakon That's actually @taraspiotr 's idea. `CATEGORY_LAYERS` array indicates, how many probability thresholds (uniformly distributed between 0 and 1) you want to use to extract objects. In case of category with index 1 (buildings) you want to have 19 thresholds: from 0.05 to 0.95 with step 0.05. That's why  `CATEGORY_LAYERS[1]==19`.

Reply:
@apyskir ok, so CATEGORY_LAYERS[0]=1 is just background threshold right?

Reply:
@jak
Yes, it's background threshold. So in some way we extract "background objects". We even extract features for them, but we drop them while preparing data for training.